controllers/administration.js

```diff
@@ -1515,6 +1515,7 @@ router.get(
 					schoolUsesLdap: res.locals.currentSchoolData.ldapSchoolIdentifier,
 					referrer: req.header('Referer'),
 					CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS,
+					alreadyHasAccount: !!account,
```

Comment:
naming, will only hasAccount not enought? sound for me more generic

Reply:
I've changed, thanks.